changelog.d/20240811_174452_chris_remove_whitelist.rst

```diff
@@ -0,0 +1,13 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Added ``Client.get_allowed_functions`` for retrieving the list of functions that are
+  allowed to be executed on an endpoint.
+
+Removed
+^^^^^^^
+
+- The ``add_to_whitelist``, ``delete_from_whitelist``, and ``get_whitelist`` functions
+  have been removed from the ``Client``. Use the ``allowed_functions`` endpoint config
+  option instead of the add/remove functions, and ``Client.get_allowed_functions``
+  instead of ``get_whitelist``.
```

Comment:
This _might_ warrant an explanation, along the lines of "uhm, why didn't y'all deprecate as is the usual practice?"  Not a big deal because these are outright broken -- so likely unused -- but may prevent a future question mark from interested parties "reading the changelog."  (But who does _that_?)

Reply:
I missed this comment before I hit approve (darn GitHub app making things harder to see), but yeah I think we should be fine - at worst we get one question in #help that should be easy to resolve